### ISSUE TITLE
Send out order via DICOM

### ISSUE THREAD

**Issue by m1kewong** [bjoernalbers/mowoli#4]
Hi,

I am new to this framework and DICOM development. I've setup the project on my mac and I can successfully create an order from the web interface. 

Now how do I send out the order to a DICOM machine? I've converted the 1.xml to a .dcm file by using xml2dcm. Do I have to use dcmsnd to send this file out?

**Comment by marcusthierfelder** [comment]
You dont, the DICOM Machine (Modality) is "asking" for the xml-file.

**Comment by m1kewong** [comment]
Thanks for your response, so I just have to setup mowoli server in the same network with the DICOM machine, create an Order (xml file) and the Modality machine would be able to read this generated 1.xml in the worklist folder under the mowoli root?

**Comment by marcusthierfelder** [comment]
Normally yes, depends a bit on the modality which you have to configure (IPAdress, the Name of Worklistserver which - if I remember correctly - has to be "MOWOLI" and the Port. The Port ist a bit tricky, because the default is 111112, which is also the default of a DICOM-Server. If you have an additional DICOM-Server for storing pictures, you have to uses different Ports)

**Comment by m1kewong** [comment]
You mean I have to setup MOWOLI to play with my existing modality server, or I have to change the setting of my current modality server in order to see MOWOLI?

**Comment by bjoernalbers** [comment]
Mowoli creates those xml-files which are served via DICOM by dcm4chee's `dcmof` tool. So you also have to start `dcmof` and pass the directory of the xml-files generated by Mowoli. Please take a look at the README's DICOM section.

**Comment by m1kewong** [comment]
I have started up mowoli server using command: bin/rails server;
After that I've started up dcmof using command: /usr/local/dcm4che/bin/dcmof --mwl /my/worklist/path MOWOLI:11112;

Then I tried to test this environment setup by requesting Worklist using a Modality Emulator from www.dvtk.org. It fails to connect to my setup.

Am I doing anything wrong or I have another tool to test my setup is correctly done?
